build_library/toolchain_util.sh

```diff
@@ -27,6 +28,9 @@ BOARD_PROFILES["amd64-usr"]="coreos:coreos/amd64/generic"
 BOARD_CHOSTS["arm64-usr"]="aarch64-cros-linux-gnu"
 BOARD_PROFILES["arm64-usr"]="coreos:coreos/arm64/generic"
 
+BOARD_CHOSTS["ppc64le-usr"]="powerpc64le-unknown-linux-gnu"
```

Comment:
Use `powerpc64le-cros-linux-gnu` for the board CHOST, it helps ensure cross-compiling works even if using a ppc64le SDK.